### PR TITLE
user profile picture and name clicks to send to osm profile

### PR DESCRIPTION
**Pull request by javtran**
resolves https://github.com/osmlab/maproulette3/issues/1757

### CODE REVIEW

File: src/pages/Leaderboard/CardLeaderboard.js [hunk 1]
```diff
@@ -22,6 +23,35 @@ class CardLeaderboard extends Component {
...
+      if (pic) {
+        pic.style.border = null
+        pic.style.outline = null
+        pic.style.outlineOffset = '-8px'
+      }
+      if(name) {
+        name.style.color = '#fff'
+      }
+    }
+
```
Comment (jschwarz2030) [inline src/pages/Leaderboard/CardLeaderboard.js:54]: DOM manipulation is a bit of an anti-pattern in React.  could you try refactoring this code using tailwind? https://tailwindcss.com/docs/hover-focus-and-other-states.  also see this https://stackabuse.com/how-to-style-hover-in-react/

Comment (javtran) [reply src/pages/Leaderboard/CardLeaderboard.js:54]: I'm looking into group-hover from tailwind. I changed the tailwind.config.js to enable group-hover, but it doesn't seem to work. Any reason for that? Are we using a version of tailwind that doesn't have group-hover?

File: src/pages/Leaderboard/CardLeaderboard.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react'
+import React, { Component} from 'react'
```
Comment (jschwarz2030) [inline src/pages/Leaderboard/CardLeaderboard.js:1]: let's keep the syntax/spacing consistent

File: src/pages/Leaderboard/CardLeaderboard.js [hunk 3]
```diff
@@ -29,16 +42,26 @@ class CardLeaderboard extends Component {
           this.props.className
         )}
       >
-        <header className="mr-max-w-xs mr-mx-auto mr-mb-2">
-          <div
-            className="mr-block mr-w-24 mr-h-24 mr-bg-black mr-bg-cover mr-bg-center mr-mx-auto mr-mb-4 mr-rounded-full"
+        <header className={"mr-max-w-xs mr-mx-auto mr-mb-2 " + (this.state.isHover ? 'hover-style': '')}>
```
Comment (jschwarz2030) [inline src/pages/Leaderboard/CardLeaderboard.js:45]: 👍

File: src/pages/Leaderboard/Leaderboard.test.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    })
+
+    test('isHover is true on hover & false on leave', () => {
+        wrapper.instance().onHover()
+        expect(wrapper.state().isHover).toEqual(true)
+
+        wrapper.instance().onLeave()
+        expect(wrapper.state().isHover).toEqual(false)
+    })
+})
```
Comment (jschwarz2030) [inline src/pages/Leaderboard/Leaderboard.test.js:58]: please use prettier on new files and stick to 2-space tabbing

**Review by javtran — COMMENTED** [review]
Thanks, will keep that in mind

**Pull request merged**